### PR TITLE
feat: init version of the github action

### PR DESCRIPTION
**Pull request by derberg**
tested it in full workflow here https://github.com/derberg/test-experiment where final output was suppose to be a html hosted on GH Pages -> https://derberg.github.io/test-experiment/

see also: https://github.com/asyncapi/asyncapi/issues/359

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: 'Generator for AsyncAPI documents'
+description: 'Use this action to generate docs or code from your AsyncAPI document. Use default templates or provide your custom ones.'
+inputs:
+  template: 
+    description: 'Template for the generator. Official templates are listed here https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate. You can pass tempalte as npm package, url to git repository, link to tar file or local template.'
```
Comment (fmvilas) [inline action.yml]:
```suggestion
    description: 'Template for the generator. Official templates are listed here https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate. You can pass template as npm package, url to git repository, link to tar file or local template.'
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
+{
+  "name": "github-action-for-generator",
+  "version": "0.0.2",
```
Comment (fmvilas) [inline package.json]: Should it be `0.1.0` or even `0.2.0`?

Comment (derberg) [reply package.json]: you are right, 0.1.0, I think I crafted it in decent way that even 1.0.0. would be fine, but generator is not 1.0 and 0.1 feels more appropriate

**Comment by derberg** [comment]
@fmvilas thanks! fixes provided

**Pull request merged**